### DISCUSSION TITLE
How to solve escaping of inline code blocks

### DISCUSSION THREAD

**Discussion by oeyoews** [vuejs/vitepress#3724] [Q&A] [answered]
https://github.com/vuejs/vitepress/issues/1767 According to this issue, I saw the relevant documents,
But if I wish to write such inline code `{{`, vitepress will also cause the same error. Is there an easier way besides using code tags and v-pre?

```md
<code v-pre>
{{
</code>
```

**Comment by brc-dd** [comment, marked as answer]
If you don't want to use vue interpolation in inline code anywhere, you can add something like this to your config:

```ts
import { defineConfig } from 'vitepress'

export default defineConfig({
  markdown: {
    config(md) {
      const defaultCodeInline = md.renderer.rules.code_inline!
      md.renderer.rules.code_inline = (tokens, idx, options, env, self) => {
        tokens[idx].attrSet('v-pre', '')
        return defaultCodeInline(tokens, idx, options, env, self)
      }
    }
  }
})
```

Then simply use it like you'd normally have:

```md
`{{ foo }}`
```

**Reply by oeyoews** [reply]
cool, I like this solution directly through custom markdown rules
